invenio_rdm_records/records/jsonschemas/records/record-v1.0.0.json

```diff
@@ -849,32 +851,48 @@
           "items": {"$ref": "#/definitions/agent"}
         },
 
-        "embargo_date": {
-          "description": "Embargo date of record (ISO8601 formatted date time in UTC). At this time both metadata and files will be made public.",
-          "type": "string",
-          "format": "date-time"
-        },
-
-        "access_condition": {
-          "description": "Conditions under which access to files are granted",
+        "embargo": {
+          "description": "Description of the embargo on the record.",
           "type": "object",
+          "required": ["active", "until", "reason"],
           "additionalProperties": false,
           "properties": {
-            "condition": {
+            "active": {
+              "description": "Whether or not the embargo is (still) active.",
+              "type": "boolean"
+            },
+            "until": {
+              "description": "Embargo date of record (ISO8601 formatted date time in UTC). At this time both metadata and files will be made public.",
               "type": "string",
-              "description": "Textual description under which conditions access is granted."
+              "format": "date-time"
```

Comment:
updated -- although the `$schema` in the jsonschema says it's draft 7, i wasn't sure if there'd be any problems (e.g. `invenio_records.validators` using a Draft4Validator)
but since the tests are running through without a problem, i guess it works